packages/common-utils/src/types.ts

```diff
@@ -466,11 +466,25 @@ export const TileTemplateSchema = TileSchema.extend({
 
 export type Tile = z.infer<typeof TileSchema>;
 
+export const DashboardFilterType = z.enum(['QUERY_EXPRESSION']);
```

Comment:
The intent here is that there may be other types of filters in the future. These filters are populated based on querying the values for an expression from ClickHouse. For example, future filters of another type my have a static list of values.